R/PipeOpTaskSurvClassifIPCW.R

```diff
@@ -0,0 +1,133 @@
+#' @title PipeOpTaskSurvClassifIPCW
+#' @name mlr_pipeops_trafotask_survclassif_IPCW
+#' @template param_pipelines
+#'
+#' @description
+#' Transform [TaskSurv] to [TaskClassif][mlr3::TaskClassif] using IPCW (Vock et al., 2016).
+#'
+#' @section Dictionary:
+#' This [PipeOp][mlr3pipelines::PipeOp] can be instantiated via the
+#' [dictionary][mlr3misc::Dictionary] [mlr3pipelines::mlr_pipeops]
+#' or with the associated sugar function [mlr3pipelines::po()]:
+#' ```
+#' PipeOpTaskSurvClassifIPCW$new()
+#' mlr_pipeops$get("trafotask_survclassif_IPCW")
+#' po("trafotask_survclassif_IPCW")
+#' ```
+#'
+#' @section Input and Output Channels:
+#' [PipeOpTaskSurvClassifIPCW] has one input channel named "input", and two
+#' output channels, one named "output" and the other "data".
+#'
+#' During training, the "output" is the "input" [TaskSurv] transformed to a
```

Comment:
this sounds confusing.

Reply:
Maybe, training the pipeline transforms the "input" [TaskSurv] to a [taskClassif.., which is the "output"